src/ui-shell/hamburger.component.ts

```diff
@@ -10,15 +10,48 @@ import { I18n } from "../i18n/i18n.module";
 	selector: "ibm-hamburger",
 	template: `
 		<button
+			(click)="clickFn($event)"
+			[ngClass]="{'bx--header__action--active': selected}"
 			class="bx--header__menu-trigger bx--header__action"
 			[attr.aria-label]="i18n.get('UI_SHELL.HEADER.MENU') | async"
 			[attr.title]="i18n.get('UI_SHELL.HEADER.MENU') | async">
-			<svg aria-hidden="true" width="20" height="20" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
+
+			<svg *ngIf="!selected" aria-hidden="true" width="20" height="20" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
 				<path d="M4 6h24v2H4zm0 18h24v2H4zm0-9h24v2H4z" />
 			</svg>
+
+			<svg *ngIf="selected" aria-hidden="true" height="20" width="20" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
```

Comment:
Is there a `@carbon/icons-angular` component we can use for this?

Reply:
As per our discussion late last week, no :(

Reply:
Gotcha, totally forgot about that 👍